akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst

```diff
@@ -165,3 +165,13 @@ Secure Cookies
 
 `Secure cookies` feature was deprecated.
 
+New Cluster Metrics Extension 
+=============================
+Previously, cluster metrics functionality was located in the ``akka-cluster`` jar.
+Now it is split out and moved into a separate akka module: ``akka-cluster-metrics`` jar.
+The module comes with few enhancements, such as use of Kamon sigar-loader 
+for native library provisioning as well as use of statistical averaging of metrics data.
+Note that both old and new metrics configuration entries in the ``reference.conf`` 
+are still in the same name space ``akka.cluster.metrics`` but are not compatible.
+Metrics extension classes and objects are located in the new package ``akka.cluster.metrics``. 
+Please see :ref:`Scala <cluster-metrics-scala>`, :ref:`Java <cluster-metrics-java>` for more information.
```

Comment:
Do you think we need to mention the most obvious migration steps for moving to the new metrics module?

Such as:
    # Disable legacy metrics in akka-cluster.
    akka.cluster.metrics.enabled=off

It might also be worth mentioning that the router config names have changed for the the module.
`cluster-metrics-adaptive-pool` and `cluster-metrics-adaptive-group`